src/main/views/task-list.njk

```diff
@@ -0,0 +1,70 @@
+{% extends "./layouts/default.layout.njk" %}
```

Comment:
Hmm, how. we. are using the macro task-list then? This seems to be hardcoding the same thing we try to achieve in macro?

Reply:
Explained offline. It's a macro I had built to pre-build a task list. It's a WIP but I don't think we should remove as we will likely need to complete this work anyway.